commit 2ced92dbd76993bd25ef900566c9a3bf881963ff
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-06-11T00:18:14+00:00 (4 months ago)
    
    MANTA-4334 binder crashing in doublecheck
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 147a359..0df70f3 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "vasync": "2.1.0",
     "verror": "^1.10.0",
     "xtend": "1.0.3",
-    "zkstream": "0.11.5"
+    "zkstream": "0.11.8"
   },
   "engines": {
     "node": ">=0.10"
